### PR TITLE
Updated the WLI module to work with new service changes

### PR DESCRIPTION
**Pull request by vanishrao**

### CODE REVIEW

File: installer/conf/omsagent.d/wlm_baseOS.conf [hunk 1]
```diff
@@ -55,3 +55,62 @@
   tag oms.wlm.monitor
 </source>
 
+<source>
```
Comment (vimish) [inline installer/conf/omsagent.d/wlm_baseOS.conf]: Shouldn't the old <source> be removed?...this conf will emit perf data twice (one for each <source> tag.)

File: installer/conf/omsagent.d/wlm_baseOS.conf [hunk 2]
```diff
@@ -55,3 +55,62 @@
...
+
+<source>
+  type wlm_oms_omi
+  object_name "Physical Disk"
+  instance_regex ".*"
+  counter_name_regex ".*"
+  interval 30s
+  omi_mapping_path %CONF_DIR_WS%/wlm_monitor_mapping.json
+  tag oms.wlm.monitor
+  wlm_ip "infrastructureInsights"
```
Comment (vimish) [inline installer/conf/omsagent.d/wlm_baseOS.conf]: <nit> i in the IP name should be in caps.

Comment (vanishrao) [reply installer/conf/omsagent.d/wlm_baseOS.conf]: This is one of the asks from the service to have it as a small "i" in the packet being sent from the Agent.

File: installer/conf/omsagent.d/wlm_baseOS.conf (hunk 1 above)
Comment (vanishrao) [reply installer/conf/omsagent.d/wlm_baseOS.conf]: The old source tags will be removed once the support for the old data type (which is the only difference between the two packets) is removed from the service.

**Pull request merged**